crates/red_knot_module_resolver/src/typeshed.rs

```diff
@@ -21,9 +37,71 @@ mod tests {
         assert!(functools_module_stub.is_file());
 
         let mut functools_module_stub_source = String::new();
-        functools_module_stub.read_to_string(&mut functools_module_stub_source)?;
+        functools_module_stub
+            .read_to_string(&mut functools_module_stub_source)
+            .unwrap();
 
         assert!(functools_module_stub_source.contains("def update_wrapper("));
-        Ok(())
+    }
+
+    #[test]
+    fn typeshed_vfs_consistent_with_vendored_stubs() {
+        let vendored_typeshed_stubs = VendoredFileSystem::new(TYPESHED_ZIP_BYTES).unwrap();
```

Comment:
We could now consider implementing `Default` where the default implementation uses the "vendored" files

Reply:
I think that would mean that `ruff_db` would have to depend on this crate, and I'd rather have it the other way around

Reply:
ah right that's true.